### ISSUE TITLE
"Add a new todo" shortcut not working on 1st start

### ISSUE THREAD

**Issue by michalhasko** [ransome1/sleek#699]
## Bug Report

**App Version:** v2.0.13

**Platform:** macOS

**Installation Method:** homebrew

**Expected Behavior:**
When pressing Cmd+N the popup to add new todo should open

**Actual Behavior:**
Cmd+N does not perform any action

**Steps to Reproduce:**
1. Open the app.
2. press Cmd+N
   - alternatively click Todos > Add new todo in the menu bar

**Notes:**
- Neither the shortcut key, nor the menu bar item to add new todo work.
- When pressing the Cmd+N shortcut, the macOS UI indicates in the same way as if the menu bar item was selected.
- Clicking the '+' button to add new todo does work.

**Comment by ransome1** [comment]
@michalhasko is it possible, that this happened after you opened sleek for the 1st time and added a todo file?

**Comment by michalhasko** [comment]
Yes, that's it. After restarting `sleek` the issue is gone. Thanks for the answer. Feel free to close the issue.
